tools/deploy/mysql_test/test_suite/issue1365/issue1365.test

```diff
@@ -0,0 +1,20 @@
+use test;
+DROP TABLE IF EXISTS numeric_table;
+create table IF NOT EXISTS numeric_table ( col1 INTEGER comment 'INTEGER', col2 INT comment 'INT', col3 SMALLINT, col4 TINYINT, col5 MEDIUMINT, col6 BIGINT, col7 DECIMAL comment 'DECIMAL', col8 NUMERIC comment 'NUMERIC', col9 FLOAT, col10 DOUBLE, col11 BIT ) DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_bin;
```

Comment:
--disable_warnings
drop table if exists t1;
--enable_warnings

for ddl statement in mysqltest, see the case example and fix your format